package.json

```diff
@@ -32,6 +32,7 @@
     "data:download:garmin": "python3 run_page/garmin_sync.py",
     "data:analysis": "python3 run_page/gen_svg.py --from-db --type github --output assets/github.svg",
     "build": "vite build",
+    "bund": "bunx --bun vite build",
```

Comment:
@frostming 
`bun` is nice. I bet you stay on top of the latest front-end tech all the time. 
We may use `bun` instead of `nodejs` in future.